hiddenlayer/sdk/services/model_scan.py

```diff
@@ -84,11 +85,14 @@ def scan_file(
                         read_amount = part.end_offset - part.start_offset
                         f.seek(int(part.start_offset))
                         part_data = f.read(int(read_amount))
-                        self._sensor_api.upload_model_part(
-                            sensor_id=sensor.sensor_id,
-                            upload_id=upload.upload_id,
-                            part=part.part_number,
+
+                        # The SaaS multipart upload returns a upload url for each part
+                        # So there is no specified route
+                        self._api_client.call_api(
+                            "PUT",
+                            part.upload_url,
                             body=part_data,
+                            header_params={"Content-Type": "application/octet-binary"},
                         )
```

Comment:
Is this true?  It makes me wonder how it worked before.

Reply:
Yeah the route now, for adhoc models, for the multipart upload is a presigned url, but before it had its own api route along with an upload id to group all the parts together. For normal models it's still a dedicated route with an upload id.

Reply:
Ah, yes, I discovered this as I switched the TypeScript SDK to use adhoc models